react/components/ProductHighlights/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl'
+import HtmlParser from 'react-html-parser'
+
+
+import styles from './styles.css'
+
+/**
+ * Product highlights Component.
+ * Render the highlights specifications of a product.
+ */
+class ProductHighlights extends Component {
+
+  render() {
+    const { highlights } = this.props
+    return (
+      <ul className={styles.content}>
+        {highlights.map((item, i) => (
+          <li key={i} className={`t-heading-5`}>{HtmlParser(item.name)}: <span className={`f5`}>{HtmlParser(item.values[0])}</span> </li>
```

Comment:
Also, have you thought whether using the index as a key is a good fit here or not? [This article](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) is a good source of knowledge